app/src/main/java/io/islnd/android/islnd/app/activities/NavBaseActivity.java

```diff
@@ -55,6 +56,8 @@
 
     private static final String TAG = NavBaseActivity.class.getSimpleName();
 
+    public static final String NOTIFICATION_ACTION = "NOTIFICATION_ACTION";
```

Comment:
I think we should precede action strings with our app's package. There are other action strings defined in `IslndIntent`. We could rename the class to `IslndAction` so it's role is more clear. I'm not sure...